app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusSheet.kt

```diff
@@ -0,0 +1,147 @@
+package org.simple.clinic.summary.teleconsultation.status
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View.NO_ID
+import com.jakewharton.rxbinding3.view.clicks
+import com.jakewharton.rxbinding3.widget.checkedChanges
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.ofType
+import kotlinx.android.synthetic.main.sheet_teleconsult_status.*
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.R
+import org.simple.clinic.ReportAnalyticsEvents
+import org.simple.clinic.di.InjectorProviderContextWrapper
+import org.simple.clinic.mobius.MobiusDelegate
+import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus.No
+import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus.StillWaiting
+import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus.Yes
+import org.simple.clinic.util.LocaleOverrideContextWrapper
+import org.simple.clinic.util.unsafeLazy
+import org.simple.clinic.util.wrap
+import org.simple.clinic.widgets.BottomSheetActivity
+import org.simple.clinic.widgets.UiEvent
+import java.util.Locale
+import java.util.UUID
+import javax.inject.Inject
+
+class TeleconsultStatusSheet : BottomSheetActivity(), TeleconsultStatusUi, TeleconsultStatusUiAction {
+
+  companion object {
+    private const val EXTRA_TELECONSULT_RECORD_ID = "teleconsultRecordId"
+
+    fun intent(
+        context: Context,
+        teleconsultRecordId: UUID
+    ): Intent {
+      return Intent(context, TeleconsultStatusSheet::class.java).apply {
+        putExtra(EXTRA_TELECONSULT_RECORD_ID, teleconsultRecordId)
+      }
+    }
+  }
+
+  @Inject
+  lateinit var locale: Locale
+
+  @Inject
+  lateinit var effectHandlerFactory: TeleconsultStatusEffectHandler.Factory
+
+  private lateinit var component: TeleconsultStatusComponent
+
+  private val teleconsultRecordId by unsafeLazy {
+    intent.getSerializableExtra(EXTRA_TELECONSULT_RECORD_ID) as UUID
+  }
+
+  private val idToTeleconsultStatus = mapOf(
```

Comment:
Can we rename the property to reflect that it is the ID of the teleconsult radio buttons instead of just `id`?